github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```
@@ -901,8 +903,17 @@ periodics:
         # login quay.io
         cat "$QUAY_PASSWORD" | docker login --username $(cat "$QUAY_USER") --password-stdin=true quay.io
 
-        # run test
-        ./automation/perfscale-test.sh
+        # run test 200
+        PERFSCALE_WORKLOAD=PERFSCALE_WORKLOAD_TWO_HUNDRED ./automation/perfscale-test.sh
```

Isn't there a `$` missing to expand the env var, or am I missing something?

If that's the case then this applies to the lines below also.
```suggestion
        PERFSCALE_WORKLOAD=$PERFSCALE_WORKLOAD_TWO_HUNDRED ./automation/perfscale-test.sh
```